site/pages/provisional-en.hbs

```diff
@@ -105,13 +105,13 @@
 		<th>CSS <code>.bg-cover</code></th>
 		<td>Background size cover</td>
 		<td>v5.1</td>
-		<td></td>
+		<td>Moved to WET-BOEW</td>
 	</tr>
 	<tr>
 		<th>Plugin <code>[data-bgimg]</code></th>
 		<td>Take the URL value and set it as the background image. To be replaced by CSS4 selector.</td>
 		<td>v5.1</td>
-		<td></td>
+		<td>Moved to WET-BOEW</td>
```

Comment:
Need a note about this deprecated markup  that the onw in wet-boew require the use of "provisional"